docs/manual/mkdocs.yml

```diff
@@ -227,15 +226,62 @@ nav:
       - project:
         - project upgrade: 'cmd/spfx/project/project-upgrade.md'
     - Microsoft Graph (graph):
+      - schemaextension:
+        - schemaextension add: 'cmd/graph/schemaextension/schemaextension-add.md'
+    - Outlook:
+        - outlook sendmail: 'cmd/outlook/outlook-sendmail.md'
+    - Microsoft Teams:
+      - teams app install: 'cmd/teams/teams-app-install.md'
```

Comment:
Articles should be grouped by the area they operate on, eg. `app`, `channel`, `funsettings`, etc. See existing `spo` commands for reference.

Reply:
Done